autoscheduler/frontend/src/components/LandingPage/SelectTerm.tsx

```diff
@@ -50,7 +52,7 @@ const SelectTerm: React.SFC = function App() {
           id="long-menu"
           anchorEl={anchorEl}
           anchorOrigin={{
-            vertical: 'bottom',
+            vertical: 'top',
```

Comment:
Honestly, I had to go digging through the Material-UI source code to fix this one, so I don't expect you to learn much from it. It's just weird.